---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Combined HIV-1 sequence and integration site analysis informs viral dynamics
  and allows reconstruction of replicating viral ancestors
authors:
- Sean C. Patro
- Leah D. Brandt
- admin
- Elias K. Halvas
- Kevin W. Joseph
- Wei Shao
- Xiaolin Wu
- Shuang Guo
- Ben Murrell
- Ann Wiegand
- Jonathan Spindler
- Castle Raley
- Christopher Hautman
- Michele Sobolewski
- Christine M. Fennessey
- Wei-Shau Hu
- Brian Luke
- Jenna M. Hasson
- Aurelie Niyongabo
- Adam A. Capoferri
- Brandon F. Keele
- Jeff Milush
- Rebecca Hoh
- Steven G. Deeks
- Frank Maldarelli
- Stephen H. Hughes
- John M. Coffin
- Jason W. Rausch
- John W. Mellors
- Mary F. Kearney
date: '2019-12-01'
doi: 10.1073/pnas.1910334116

# Schedule page publish date (NOT publication's date).
publishDate: '2022-04-26T15:49:39.707961Z'

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types:
- '2'

# Publication name and optional abbreviated publication name.
publication: '*Proceedings of the National Academy of Sciences*'
publication_short: ''

abstract: Significance To develop a cure for HIV-1, it is necessary to understand
  how infected cells persist despite treatment. Integrated HIV DNA (proviruses) can
  be distinguished by their sites of integration into the host genome and by their
  proviral sequence. We applied multiple-displacement amplification (MDA) to single
  proviruses isolated from blood and lymph nodes to determine their integration sites
  and full-length sequences. We found that identical subgenomic HIV-1 sequences can
  result from either clonal expansion or from genetic bottlenecks that occurred with
  transmission or with the emergence of drug resistance mutations. Furthermore, we
  show this MDA approach can be used to infer intact ancestral HIV-1 sequences from
  the archives of defective ones, providing an approach for investigations of HIV-1
  evolution in vivo. ,  Understanding HIV-1 persistence despite antiretroviral therapy
  (ART) is of paramount importance. Both single-genome sequencing (SGS) and integration
  site analysis (ISA) provide useful information regarding the structure of persistent
  HIV DNA populations; however, until recently, there was no way to link integration
  sites to their cognate proviral sequences. Here, we used multiple-displacement amplification
  (MDA) of cellular DNA diluted to a proviral endpoint to obtain full-length proviral
  sequences and their corresponding sites of integration. We applied this method to
  lymph node and peripheral blood mononuclear cells from 5 ART-treated donors to determine
  whether groups of identical subgenomic sequences in the 2 compartments are the result
  of clonal expansion of infected cells or a viral genetic bottleneck. We found that
  identical proviral sequences can result from both cellular expansion and viral genetic
  bottlenecks occurring prior to ART initiation and following ART failure. We identified
  an expanded T cell clone carrying an intact provirus that matched a variant previously
  detected by viral outgrowth assays and expanded clones with wild-type and drug-resistant
  defective proviruses. We also found 2 clones from 1 donor that carried identical
  proviruses except for nonoverlapping deletions, from which we could infer the sequence
  of the intact parental virus. Thus, MDA-SGS can be used for “viral reconstruction”
  to better understand intrapatient HIV-1 evolution and to determine the clonality
  and structure of proviruses within expanded clones, including those with drug-resistant
  mutations. Importantly, we demonstrate that identical sequences observed by standard
  SGS are not always sufficient to establish proviral clonality.

# Summary. An optional shortened abstract.
summary: ''

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf:
url_code: https://github.com/michaelbale/prob_identical
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ''
links:
- name: URL
  url: https://pnas.org/doi/full/10.1073/pnas.1910334116
- name: Shiny App
  url: https://michaelbale.shinyapps.io/prob_identical/
  icon_pack: fab
  icon: browser
---
