---
# Documentation: https://wowchemy.com/docs/managing-content/

title: An analytical pipeline for identifying and mapping the integration sites of
  HIV and other retroviruses
authors:
- Daria W. Wells
- Shuang Guo
- Wei Shao
- admin
- John M. Coffin
- Stephen H. Hughes
- Xiaolin Wu
date: '2020-03-01'
doi: 10.1186/s12864-020-6647-4

# Schedule page publish date (NOT publication's date).
publishDate: '2022-04-26T15:49:38.348622Z'

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types:
- '2'

# Publication name and optional abbreviated publication name.
publication: '*BMC genomics*'
publication_short: ''

abstract: 'BACKGROUND: All retroviruses, including human immunodeficiency virus (HIV),
  must integrate a DNA copy of their genomes into the genome of the infected host
  cell to replicate. Although integrated retroviral DNA, known as a provirus, can
  be found at many sites in the host genome, integration is not random. The adaption
  of linker-mediated PCR (LM-PCR) protocols for high-throughput integration site mapping,
  using randomly-sheared genomic DNA and Illumina paired-end sequencing, has dramatically
  increased the number of mapped integration sites. Analysis of samples from human
  donors has shown that there is clonal expansion of HIV infected cells and that clonal
  expansion makes an important contribution to HIV persistence. However, analysis
  of HIV integration sites in samples taken from patients requires extensive PCR amplification
  and high-throughput sequencing, which makes the methodology prone to certain specific
  artifacts. RESULTS: To address the problems with artifacts, we use a comprehensive
  approach involving experimental procedures linked to a bioinformatics analysis pipeline.
  Using this combined approach, we are able to reduce the number of PCR/sequencing
  artifacts that arise and identify the ones that remain. Our streamlined workflow
  combines random cleavage of the DNA in the samples, end repair, and linker ligation
  in a single step. We provide guidance on primer and linker design that reduces some
  of the common artifacts. We also discuss how to identify and remove some of the
  common artifacts, including the products of PCR mispriming and PCR recombination,
  that have appeared in some published studies. Our improved bioinformatics pipeline
  rapidly parses the sequencing data and identifies bona fide integration sites in
  clonally expanded cells, producing an Excel-formatted report that can be used for
  additional data processing. CONCLUSIONS: We provide a detailed protocol that reduces
  the prevalence of artifacts that arise in the analysis of retroviral integration
  site data generated from in vivo samples and a bioinformatics pipeline that is able
  to remove the artifacts that remain.'

# Summary. An optional shortened abstract.
summary: ''

tags:
- Humans
- Chromosome Mapping
- Computational Biology
- DNA
- Viral
- Genome
- Human
- HIV
- HIV Infections
- Integration
- Patient samples
- PCR mispriming
- PCR recombination
- Polymerase Chain Reaction
- Proviruses
- Retrovirus
- Sequence Analysis
- DNA
- Virus Integration
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf:
url_code: https://github.com/HughesLab-FNL/ISA-Analysis-Pipeline
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ''

links:
- name: URL
  url: https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6647-4
---
